ci/integration.yml

```diff
@@ -0,0 +1,31 @@
+---
+resources:
+  - name: secrets
+    type: s3-iam
+    source:
+      bucket: {{state_bucket}}
+      region_name: {{aws_region}}
+      versioned_file: {{secrets_file}}
+
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.8-alpine
+inputs:
+  - name: pr
+  - name: secrets
+run:
+  path: sh
+  args:
+    - -e
+    - -c
+    - |
+      export ACCESS_TOKEN=$(awk -F':' '$1 ~ /compose_access_token/ {print $2}' ./secrets/compose-broker-secrets.yml)
```

Comment:
> This is also not very robust

I agree, but I'm not sure it matters given the complexity of any alternatives.

> as it won't cope with the value being quoted in the YAML file (which it can be depending on the contents)

I'm uncertain. Could you show a code example?